frontend/package.json

```diff
@@ -20,7 +20,7 @@
     "test:e2e:ci": "vue-cli-service test:e2e --headless",
     "test:unit:debug": "node --inspect-brk=0.0.0.0:9229 ./node_modules/@vue/cli-service/bin/vue-cli-service.js test:unit --no-cache --runInBand",
     "test:unit:watch": "vue-cli-service test:unit --watch",
-    "postinstall": "sed -i 's/Color.extend(contrast);/Color.extend({contrast});/g' node_modules/colorjs.io/dist/color.js"
+    "postinstall": "sed 's/Color.extend(contrast);/Color.extend({contrast});/g' node_modules/colorjs.io/dist/color.js > node_modules/colorjs.io/dist/color.js.tmpfix; cat node_modules/colorjs.io/dist/color.js.tmpfix > node_modules/colorjs.io/dist/color.js; rm node_modules/colorjs.io/dist/color.js.tmpfix"
```

Comment:
The `npm install` command should be executed inside a docker container, which should ensure that the correct version of `sed` including support for the `-i` flag should be present. In my opinion, at this point, it is not a supported setup to run `npm install` directly on your mac. Do you disagree?

Reply:
The issue is not coming from running this natively, but when running the docker containers via normal `docker-compose up`. As far as I could figure out, it seems to be an edge case involving VirtioFS and a bug in sed < 4.8 (https://forums.docker.com/t/sed-couldnt-open-temporary-file-xyz-permission-denied-when-using-virtiofs/125473/4)

For sure we could try upgrading sed inside the containers. But assuming this is anyway a temporary hack and if above still works for other platforms, this looks good enough to me.